composables/rotki-sponsorship/payment.ts

```diff
@@ -182,11 +183,44 @@ export function useRotkiSponsorshipPayment() {
       const receipt = await tx.wait();
 
       if (receipt?.status === 1) {
+        // Parse the NFTMinted event to get the token ID
+        let tokenId: string | undefined;
+
+        const { CONTRACT_ADDRESS: contractAddress } = useNftConfig();
```

Comment:
shouldn't we call the composable in the context of the other composable just to be safe?